matching-core/pom.xml

```diff
@@ -40,9 +40,9 @@
             <version>${log4j.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.graphhopper.external</groupId>
+            <groupId>com.bmw.hmm</groupId>
```

Comment:
As the hmm lib is included in this repository, I would prefer to change the groupId there and here to com.graphhopper.external, so that we can publish the lib under maven central (until you get approval)